### PR TITLE
Various Non Semantic Fixes

### PR DESCRIPTION
**Pull request by liam-middlebrook**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [X] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

A collection of non-semantic fixes that mostly deal with whitespace.

### CODE REVIEW

**Comment by liam-middlebrook** [comment]
Can another @ComputerScienceHouse/constitution-maintainers look at this?

**Comment by TalCohen** [comment]
Seems totally fine to me. Did you just want another set of eyes on it just to be sure, or did we change the non-semantic change process?

File: articles.tex [hunk 1]
```diff
@@ -116,11 +116,11 @@
 \asubsection{Maintainer Expectations}
 Maintainers are expected to:
 \begin{itemize}
-    \item Review changes to the Constitution for grammar, spelling, and internal consistency.
-    \item Keep a public record of changes to the Constitution.
-    \item Participate in discussion of proposals.
-    \item Facilitate the creation of new changes to the Constitution by other members.
-    \item Be knowledgeable about the Constitution.
+	\item Review changes to the Constitution for grammar, spelling, and internal consistency.
```
Comment (gambogi) [inline articles.tex]: Is this indentation style consistent with the rest of the document or only the lines you touched?

**Comment by liam-middlebrook** [comment]
Even though we can arbitrarily make non semantic changes I prefer to have them reviewed by another maintainer so that people will do so in the future.

**Comment by TalCohen** [comment]
👍🏻 ship it!

**Comment by gambogi** [comment]
@bmbowdish did the question I had get answered?

File: articles.tex (hunk 1 above)
Comment (bmbowdish) [inline articles.tex]: Yes it is.

**Pull request merged**